main/src/main/scala/sbt/internal/CommandExchange.scala

```diff
@@ -97,22 +92,11 @@ private[sbt] final class CommandExchange {
    * Check if a server instance is running already, and start one if it isn't.
    */
   private[sbt] def runServer(s: State): State = {
-    lazy val port = (s get serverPort) match {
-      case Some(x) => x
-      case None    => 5001
-    }
-    lazy val host = (s get serverHost) match {
-      case Some(x) => x
-      case None    => "127.0.0.1"
-    }
-    lazy val auth: Set[ServerAuthentication] = (s get serverAuthentication) match {
-      case Some(xs) => xs
-      case None     => Set(ServerAuthentication.Token)
-    }
-    lazy val connectionType = (s get serverConnectionType) match {
-      case Some(x) => x
-      case None    => ConnectionType.Tcp
-    }
+    lazy val port = s get serverPort getOrElse 5001
```

Comment:
Subjective: The chained infix I think is harder to read compared to `s.get.serverPort.getOrElse(5001)`.